doc/tutorials/mlip/mlip.ipynb

```diff
@@ -0,0 +1,447 @@
+{
```

Comment:
<p>Why not show it?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/espressomd/espresso/pull/4982/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>